plugins/modules/cloud/misc/proxmox_nic.py

```diff
@@ -0,0 +1,334 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Lammert Hellinga (@Kogelvis) <lammert@hellinga.it>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: proxmox_nic
+short_description: Management of NIC's for Qemu(KVM) VM's in a Proxmox VE cluster.
+description:
+  - Allows you to create/update/delete NIC's on Qemu(KVM) Virtual Machines in a Proxmox VE cluster.
+author: "Lammert Hellinga (@Kogelvis) <lammert@hellinga.it>"
+options:
+  bridge:
+    description:
+      - Add this interface to the specified bridge device. The Proxmox VE standard bridge is called 'vmbr0'
+    type: str
+  firewall:
+    description:
+      - Whether this interface should be protected by the firewall.
+    type: bool
+    default: False
+  interface:
+    description:
+      - Name of the interface, should be C(net[n]) where C(1 ≤ n ≤ 31)
+    type: str
+    required: True
+  link_down:
+    description:
+      - Whether this interface should be disconnected (like pulling the plug).
+    type: bool
+    default: False
+  mac:
+    description:
+      - C(XX:XX:XX:XX:XX:XX) should be a unique MAC address. This is automatically generated if not specified.
+      - When not specified this module will keep the MAC address the same when changing an existing interface.
+    type: str
+  model:
+    description:
+      - Model is one of C(e1000 e1000-82540em e1000-82544gc e1000-82545em i82551 i82557b i82559er ne2k_isa ne2k_pci pcnet rtl8139 virtio vmxnet3)
+    type: str
+    choices: ['e1000', 'e1000-82540em', 'e1000-82544gc', 'e1000-82545em', 'i82551', 'i82557b', 'i82559er', 'ne2k_isa', 'ne2k_pci', 'pcnet',
+              'rtl8139', 'virtio', 'vmxnet3']
+    default: virtio
+  mtu:
+    description:
+      - Force MTU, for C(virtio) model only. Set to '1' to use the bridge MTU
+      - Value should be C(1 ≤ n ≤ 65520)
+    type: int
+  name:
+    description:
+      - Specifies the VM name. Only used on the configuration web interface.
+      - Required only for C(state=present).
+    type: str
+  queues:
+    description:
+      - Number of packet queues to be used on the device.
+      - Value should be C(0 ≤ n ≤ 16)
```

Comment:
```suggestion
      - Value should be C(0 ≤ n ≤ 16).
```
Same for others. (Every sentence should end with a period.)